low-level/files/opt/seagate/sspl/setup/resource_map/__init__.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/python3
+#!/usr/bin/python3.6
```

Comment:
We can create soft links for required python bin reference
> ln -sf /usr/bin/python3.6 /bin/python3

instead of changing shebang all over

Reply:
Okay, let me check if this is possible. 👍

Reply:
```
╙──▶ ll /bin
lrwxrwxrwx. 1 root root 7 Mar 14 13:28 /bin -> usr/bin
```
This soft link already exists @cdeshmukh still, the build was failing with such shebang.

Reply:
try to find solutions in online forums, there will be some solution, why el8 would break back compatibility, for shebang usage ?

Reply:
In el8, installation in /bin is not supported, /bin already has a soft-link to /usr/bin, but at build time while installing the rpms /bin path in the shebang is not detected and installation fails, so using /usr/bin in shebang is necessary for el8  systems.
https://dnf.readthedocs.io/en/latest/cli_vs_yum.html#dnf-provides-bin-file-is-not-fully-supported
<img src="https://user-images.githubusercontent.com/45289079/127119166-070650c2-4592-4f20-8d63-607d094a6578.png" width="600">

Reply:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/considerations_in_adopting_rhel_8/index#python_dynamic-programming-languages-web-servers-database-servers